app-ios/Sources/SearchFeature/SelectButtonView.swift

```diff
@@ -0,0 +1,55 @@
+import Assets
+import Model
+import SwiftUI
+import Theme
+
+struct SelectButtonView: View {
+    let title: String
+    let selected: Bool
+    let onSelect: @Sendable () -> Void
+    let onDeselect: @Sendable () -> Void
+    var body: some View {
+        HStack(spacing: 20) {
+            Button {
+                selected ? onDeselect() : onSelect()
+            } label: {
+                selected ?
+                    Image(systemName: "checkmark.square.fill")
+                        .resizable()
+                        .frame(width: 18, height: 18)
+                        .foregroundColor(AssetColors.primary.swiftUIColor)
+                    :
+                    Image(systemName: "square")
+                        .resizable()
+                        .frame(width: 18, height: 18)
+                        .foregroundColor(AssetColors.primary.swiftUIColor)
```

Comment:
How about use `Group` ?
This can be converted to following(example).

```swift
Group {
  selected 
    ? Image(...)
        .resizable()
    : Image(...)
        .resizable()
}
.frame(width: 18, height: 18)
.foregroundColor(AssetColors.primary.swiftUIColor)
```